app/cp_nigeria/views.py

```diff
@@ -450,36 +410,44 @@ def cpn_model_choice(request, proj_id, step_id=6):
         "step_list": CPN_STEP_VERBOSE,
     }
 
-    html_template = "cp_nigeria/steps/scenario_step6.html"
-
-    qs_bm = BusinessModel.objects.filter(scenario=project.scenario)
+    html_template = "cp_nigeria/steps/scenario_model_choice.html"
 
     if request.method == "GET":
-        # html_template = "cp_nigeria/steps/scenario_step6.html"
-        score = None
-        if qs_bm.exists():
-            bm = qs_bm.get()
-            score = bm.total_score
-        context["form"] = ModelSuggestionForm(score=score)
-        context["score"] = score
-        if score is not None:
-            context["recommanded_model"] = model_score_mapping(score)
-    return render(request, html_template, context)
+        bm, created = BusinessModel.objects.get_or_create(
+            scenario=project.scenario, defaults={"scenario": project.scenario}
+        )
+        context["form"] = ModelSuggestionForm(instance=bm)
+        context["bm"] = bm
+        context["score"] = bm.total_score
+
+        recommended = context["form"].fields["model_name"].initial
+        if recommended is not None:
+            context["recommanded_model"] = recommended
```

Comment:
```suggestion
            context["recommanded_model"] = recommended
            context["form"] = ModelSuggestionForm(instance=bm, initial={"model_name": recommended})
```

load the recommended business model as the initial value to the form, so the information for the suggested business models gets loaded immediately after saving the questionnaire (without having to select it from the drop down)